media/src/utils.js

```diff
@@ -27,11 +27,31 @@ const joinUrl = function(base, path) {
     return url.href;
 };
 
+/**
+ * Converts parameters and objects into URL Search Parameters
+ * @param {*} params 
+ * @param {*} objects 
+ * @returns URLSearchParams
+ */
+const passObjects = function(params, objects) {
```

Comment:
A function name like `convertToURLParams()` would be more descriptive here. Also, isn't all the data we need in the `objects` array? It's not clear what the `params` input is doing here.

Reply:
Params was included to allow it to be used by the `Encode URL` feature which took into account Chapter selection, menu and grid visibility, and whether it was looking at Chapters or the Object List.